envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/SnapshotProperties.kt

```diff
@@ -166,6 +166,7 @@ class TrafficSplittingProperties {
     var zoneName = ""
     var headerName = ""
     var weightsByService: Map<String, ZoneWeights> = mapOf()
+    var zonesAllowingTrafficSplitting = listOf<String>()
```

Comment:
does defining it like this allow for using both `zonesAllowingTrafficSplitting` and `zones-allowing-traffic-splitting`? 
I see that the properties are used both ways in [config PRs](https://github.com/allegro-internal/alle-envoy-control-config/commit/d4d26a7ba87be935b43086ef659a8e2df8e95529#diff-438d442126cd51412e0df44a27f72c0aaec0690a0c93af109301c39e4411a9eeL130), other properties in [this PR](https://github.com/allegro/envoy-control/pull/414/files#diff-4c14cffe1498e7d00744dffea81106f21fce5b1b9c4d20524364006d0d949fa8R25)
Traffic splitting tests use [zonesAllowingTrafficSplitting](https://docs.google.com/document/d/12izmBIDbSkeVLfrXltxjG1zpPkA2PAI8kURakwjwGi0/edit#bookmark=id.9mju8widuelu), while docs have [zones-allowing-traffic-splitting](https://docs.google.com/document/d/1KVmEu3Cqi9E4TI4-ot8CFVyxjblTi69gQWiHaxjRYEI/edit#heading=h.fiidnpxsl1ge)

also pls add to `configuration.md` and while you are at it I think `envoy-control.envoy.snapshot.load-balancing.traffic-splitting.weights-by-service-properties` should be fixed to `envoy-control.envoy.snapshot.load-balancing.traffic-splitting.weights-by-service`

Reply:
so it's allowed both ways, in all aec configs mostly used kebab case, so I added examples to the docs similarly, in the tests I was just copying from properties so it's camel case, I fixed some properties in tests to make them consistent throughout the test, but I can fix to kebab case as well

Reply:
and fixed property in configuration.md